.changeset/mean-chefs-learn.md

```diff
@@ -0,0 +1,15 @@
+---
+"@ignored/hardhat-vnext-viem": major
```

Comment:
`hardhat-viem` is the only new package, everything else has been released before.